src/pysdmx/api/gds/__init__.py

```diff
@@ -151,36 +151,30 @@ def get_catalogs(
         """Get the list of catalogs for the supplied parameters.
 
         Args:
-                catalog: The agency maintaining the catalog.
-                resource: The resource ID(s) to query. Defaults to '*'.
-                version: The version(s) of the resource. Defaults to '*'.
-                resource_type: Filters the endpoints that support
-                 the requested resource type (eg, 'data', 'metadata')
-        .
-                message_format: Filters the endpoints that support any
-                  of the requested message formats.
-
-                api_version: Filters the endpoints that is in a
-                  specific SDMX API version.
-                  Multiple values separated by commas are possible.
-                  By default (if nothing is sent) it returns everything.
-
-                detail: The amount of information to be returned.
-                    Option full: All available information for all artefacts
-                      should be returned.
-                    Option raw: Any nested service will be referenced.
-
-                references: Instructs the web service to return
-                  (or not) the artefacts referenced by the
-                  artefact to be returned.
-
-                    Option none: No referenced artefacts will be returned.
-                    Option children: Returns the artefacts
-                      referenced by the artefact to be returned.
-
+            catalog: The agency maintaining the catalog.
+            resource: The resource ID(s) to query. Defaults to '*'.
+            version: The version(s) of the resource. Defaults to '*'.
```

Comment:
Is that so? In SDMX-REST, by default, you don't get all versions but only the latest one...